Doc/faq/extending.rst

```diff
@@ -254,146 +254,6 @@ For Red Hat, install the python-devel RPM to get the necessary files.
 
 For Debian, run ``apt-get install python-dev``.
 
-
-How do I tell "incomplete input" from "invalid input"?
```

Comment:
Wait, we are removing a lot here. There is still the parts about `codeop` and `PyRun_InteractiveLoop` that are useful. We need to remove from this part forward:

>However sometimes you have to run the embedded Python interpreter in the same thread as your rest application and you can't allow the...

Reply:
@pablogsal Ha! Clearly, I've gone too eagerly too far. I've restored the relevant part. Thank you.